cthreads.h

```diff
@@ -446,17 +446,28 @@ int cthreads_cond_timedwait(struct cthreads_cond *cond, struct cthreads_mutex *m
   int cthreads_error_code(void);
 #endif
 
+#if ((defined(__GNUC__) || defined(__clang__) || defined(__TINYC__)) && __STDC__ == 1 && __STDC_VERSION__ >= 199901L) \
+    || (__STDC__ == 0 && __STDC_VERSION__ >= 201112L)
+#define CTHREADS_STATIC_VLA 1
+#else
+#define CTHREADS_STATIC_VLA 0
```

Comment:
```suggestion
  #define CTHREADS_STATIC_VLA 0
```

For the one below too. And if applied the suggestion above, there's no need for #else.

Reply:
I applied the earlier suggestion, so I'll resolve this.